tpot/base.py

```diff
@@ -141,10 +141,14 @@ def __init__(self, generations=100, population_size=100, offspring_size=None,
             Random number generator seed for TPOT. Use this to make sure
             that TPOT will give you the same results each time you run it
             against the same data set with that seed.
-        config_dict: dictionary (default: None)
-            Configuration dictionary for customizing the operators and parameters that
-            TPOT uses in the optimization process.
-            For examples, see config_regressor.py and config_classifier.py
+        config_dict: dictionary or string (default: None)
+            Dictionary:
```

Comment:
Please replace "dictionary" with "file path", and describe it as a file path to a configuration dictionary.